### ISSUE TITLE
Transactions generated via dApp wallet connector should have TTL

### ISSUE THREAD

**Issue by agaffney** [TosiDrop/vm-frontend#139]
When connecting a wallet (such as Eternl) and using the "Send Ada" button on the claims page, the generated transaction that I'm prompted to sign shows as `no time limit`. A transaction generated from within Eternl will show a time limit of 3 hours by default.

**Comment by wolf31o2** [comment]
From upstream examples:
```
// set the time to live - the absolute slot value before the tx becomes invalid
txBuilder.set_ttl(410021);
```

**Comment by aryelciu001** [comment]
I'll use @wolf31o2 code above for the tx signing

**Comment by wolf31o2** [comment]
I added an endpoint to get the current absolute slot, too.

**Comment by reqlez** [comment]
@aryelciu001 is this completed? Can you go thru the issues list and close the ones you know have been completed?

**Comment by wolf31o2** [comment]
Closed by #345
